extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -238,12 +239,25 @@ export class ConsentUI {
    * @return {!Element}
    */
   createPlaceholder_() {
-    // TODO(@zhouyx): Allow publishers to provide placeholder upon request
     const placeholder = this.parent_.ownerDocument.createElement('placeholder');
     toggle(placeholder, false);
     const {classList} = placeholder;
-    classList.add(consentUiClasses.fill);
-    classList.add(consentUiClasses.placeholder);
+    classList.add(consentUiClasses.loadingPlaceholder);
```

Comment:
Since you're only `add`ing a single class to the `classList`, you can remove the line above and just call directly.

```suggestion
    placeholder.classList.add(consentUiClasses.loadingPlaceholder);
```